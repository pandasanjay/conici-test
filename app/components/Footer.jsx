import React from "react";
import "./Footer.scss";
import Conichi from "../assets/conichi.png";
import PartnerLogo from "../assets/partner-logo.png";
const Footer = function({ copywrite }) {
  return (
    <div>
      <div class="d-flex justify-content-between pt-5">
        <div className="pb-5 pl-5">
          <img src={Conichi} data-selector="conichi-logo" />
        </div>
        <div className="pb-5 pr-5">
          <img src={PartnerLogo} data-selector="partner-logo" />
        </div>
      </div>
      <footer className="navbar navbar-light bg-light sh-footer">
        <div />
        <div className="sh-copy-write p-3">{copywrite}</div>
        <div />
      </footer>
    </div>
  );
};

export default Footer;
