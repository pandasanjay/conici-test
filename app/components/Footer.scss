@import "../var.scss";

.sh-footer {
    .sh-copy-write {
        font-family: $font-family;
        font-size: 12px;
    }

}

.sh-company-logo {
    width: 86px;
    height: 25px;
    background-image: url("./../assets/conichi.png");
    background-position: center;
    background-repeat: no-repeat;
}

.sh-partner-logo {
    width: 148px;
    height: 25px;
    background-image: url("./../assets/partner-logo.png");
    background-position: center;
    background-repeat: no-repeat;
}