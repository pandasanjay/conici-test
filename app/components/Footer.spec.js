import React from "react";
import Footer from "./Footer";
import { shallow } from "enzyme";
describe("Footer component", () => {
  let component;
  beforeEach(() => {
    component = shallow(<Footer copywrite="@ 2018 - Hotel Beacons GmbH" />);
  });
  it("should have match with snapshot", () => {
    expect(component).toMatchSnapshot();
  });
});
