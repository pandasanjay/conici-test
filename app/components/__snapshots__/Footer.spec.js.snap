// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should have match with snapshot 1`] = `
<div>
  <div
    className="d-flex justify-content-between pt-5"
  >
    <div
      className="pb-5 pl-5"
    >
      <img
        data-selector="conichi-logo"
        src="test-file-stub"
      />
    </div>
    <div
      className="pb-5 pr-5"
    >
      <img
        data-selector="partner-logo"
        src="test-file-stub"
      />
    </div>
  </div>
  <footer
    className="navbar navbar-light bg-light sh-footer"
  >
    <div />
    <div
      className="sh-copy-write p-3"
    >
      @ 2018 - Hotel Beacons GmbH
    </div>
    <div />
  </footer>
</div>
`;
